client/app/components/dynamic-table/index.js

```diff
@@ -34,11 +34,11 @@ function sortRows(rows, orderBy) {
     for (let i = 0; i < orderBy.length; i += 1) {
       va = a[orderBy[i].name];
       vb = b[orderBy[i].name];
-      if (va < vb) {
+      if (!va || (va < vb)) {
         // if a < b - we should return -1, but take in account direction
         return orderBy[i].direction * -1;
       }
-      if (va > vb) {
+      if ((va > vb) || !vb) {
```

Comment:
I think you need explicit `null`/`undefined` check here instead of `!vb`/`!va`, because `!0 == !'' == true` in JavaScript...